frontend/public/components/factory/list.jsx

```diff
@@ -327,9 +328,17 @@ export const List = connect(stateToProps, {sortList: UIActions.sortList})(
       <Rows key="rows" expand={expand} Row={Row} {...componentProps} />
     </React.Fragment>;
 
-    return <div className="co-m-table-grid co-m-table-grid--bordered">
-      { fake ? children : <StatusBox {...props}>{children}</StatusBox> }
-    </div>;
+    const content = fake ? children : <StatusBox {...props}>{children}</StatusBox>;
+
```

Comment:
For the record, we considered doing `<List responsive={true} {...props} />` vs `<ResponsiveList><List {...props} /></ResponsiveList />`.  I think this single attribute addition works fine.  Its possible we would want to revisit if we keep adding props in the future.